resources/lang/en/validation.php

```diff
@@ -12,82 +10,80 @@
     | as the size rules. Feel free to tweak each of these messages here.
     |
     */
-
-    'accepted'               =>   'The :attribute must be accepted.',
-    'active_url'             =>   'The :attribute is not a valid URL.',
-    'after'                  =>   'The :attribute must be a date after :date.',
-    'after_or_equal'         =>   'The :attribute must be a date after or equal to :date.',
-    'alpha'                  =>   'The :attribute may only contain letters.',
-    'alpha_dash'             =>   'The :attribute may only contain letters, numbers, and dashes.',
-    'alpha_num'              =>   'The :attribute may only contain letters and numbers.',
-    'array'                  =>   'The :attribute must be an array.',
-    'before'                 =>   'The :attribute must be a date before :date.',
-    'before_or_equal'        =>   'The :attribute must be a date before or equal to :date.',
-    'between'                =>   [
-        'numeric'   =>   'The :attribute must be between :min and :max.',
-        'file'      =>   'The :attribute must be between :min and :max kilobytes.',
-        'string'    =>   'The :attribute must be between :min and :max characters.',
-        'array'     =>   'The :attribute must have between :min and :max items.',
+    'accepted'              =>  'The :attribute must be accepted.',
```

Comment:
Convention: trước và sau dấu `=>` là 1 space.
ví dụ `'accepted'  =>  'The :attribute must be accepted.',`
-->Fix cả file

Reply:
chưa fix